Dockerfile

```diff
@@ -0,0 +1,9 @@
+# https://docs.docker.com/compose/rails/
+FROM ruby:2.2.0
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
I believe `apt-transport-https` and `lsb-release` are not required, but give it a try ;)